### PR TITLE
Add isRuleComplete tests and fix bug

### PR DESCRIPTION
**Pull request by norkans7**
follow up to #684

### CODE REVIEW

**Comment by norkans7** [comment]
@ericnewcomer please review

File: static/coffee/flows/controllers.coffee [hunk 1]
```diff
@@ -1105,7 +1105,7 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
     else if rule._config.operands == 1 and not rule.test._base
       complete = false
 
-    else if rule._config.type == 'between' and (not rule.test.min or not rule.test.min)
+    else if rule._config.type == 'between' and (not rule.test.min or not rule.test.max)
```
Comment (norkans7) [inline static/coffee/flows/controllers.coffee]: And there were a bug here

Comment (ericnewcomer) [inline static/coffee/flows/controllers.coffee]: Nice! Hooray for unit tests!

**Comment by ericnewcomer** [comment]
🎉

**Pull request merged**